<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slim Unlock for Huananzhi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Slim Unlock for Huananzhi&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>521</width>
      <height>120</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainL">
     <item>
      <layout class="QHBoxLayout" name="parserL">
       <item>
        <widget class="QComboBox" name="siteInfoCmb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="duplicatesEnabled">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>Требуется обновление / Needs update</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateSiteInfoBtn">
         <property name="text">
          <string>Обновить/Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="openFileBtn">
         <property name="text">
          <string>Открыть / Open</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="saveL">
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Сохранить/Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="patchBtn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Пропатчить / Patch</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="dwnBiosPb">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mmtoolCb">
         <property name="text">
          <string>Удаление микрокода / Removing microcode</string>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="amiCb">
         <property name="text">
          <string>Установка настроек / Setting the settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
